Dockerfile

```diff
@@ -1,14 +1,11 @@
-FROM alpine:3.4
+FROM node:8.1-alpine
```

Comment:
Is getting to use Node 8 the reason you want to use the Node image? The Node image is twice as large as installing Node directly on Alpine.